layouts/shortcodes/skew.html

```diff
@@ -57,8 +65,10 @@
 <!--
 example shortcode use:
 - skew nextMinorVersion
-- skew latestVersion
+- skew currentVersion
 - skew prevMinorVersion
 - skew oldestMinorVersion
+- skew latestVersion
+- skew latestVersionAddMinor -1 "-"
```

Comment:
latestVersionAddMinor is only used in the kubeadm upgrade page and it should be based on `current` - i.e. `currentVersionAddMinor`

Reply:
- I'm also using this new shortcode on the page about supported versions and skew.
- Other localizations remain using the old shortcode.